### PR TITLE
[NP-10472] time of day schedule time zone support

### PR DESCRIPTION
**Pull request by jinnnn9**
https://nanopay.atlassian.net/browse/NP-10472

### CODE REVIEW

File: src/foam/nanos/cron/TimeOfDaySchedule.js [hunk 1]
```diff
@@ -21,10 +21,17 @@ foam.CLASS({
...
+    'java.time.*',
+    'static foam.util.DateUtil.*',
   ],
 
   properties: [
+    {
+      class: 'Reference',
+      of: 'foam.time.TimeZone',
+      name: 'timeZone',
+      value: 'Africa/Abidjan' // UTC/GMT
```
Comment (kgrgreer) [inline src/foam/nanos/cron/TimeOfDaySchedule.js]: Wouldn't just 'GMT' for a clearer choice?

Comment (jlhughes) [reply src/foam/nanos/cron/TimeOfDaySchedule.js]: GMT doesn't exist.

Comment (jlhughes) [reply src/foam/nanos/cron/TimeOfDaySchedule.js]: The label in the TimeZone DAO has GMT in it.

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/cron/TimeOfDaySchedule.js (hunk 1 above)
Comment (jlhughes) [reply src/foam/nanos/cron/TimeOfDaySchedule.js]: An option is a duplicate entry.

**Pull request merged**